CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to the Conjur API for Java
+
+## Prerequisites
+### Java
+To work in this codebase, you will want to need the latest version of the Java JDK installed. Dependencies for this project will be installed locally when the project is built.
+
+### Docker
+This project is container-based and therefore, you will need [Docker](https://docs.docker.com/v17.12/) in order to make and view your changes.
+
+### Access to Conjur Registry
+This project requires both OSS and Enterprise images. You shouldn't have a problem pulling down the OSS image, but to pull the Enterprise image, you must be granted access to the Conjur registry. 
+
+You will want to get in contact with the Infrastructure team via our Slack to be granted access. If you have been granted access, run: `docker login registry2.itci.conjur.net` 
```

Comment:
Can you link to our public slack? is there a specific channel?

Reply:
sure!